pkg/store/tsdb.go

```diff
@@ -24,6 +23,8 @@ import (
 	"github.com/thanos-io/thanos/pkg/store/storepb"
 )
 
+const RemoteReadFrameLimit = 1048576
```

Comment:
Moved here from `testutil` package to remove the `testutil` dependency from `pkg/store` (excluding tests), which is the reason why -- this time -- we can't upgrade Prometheus in Cortex without upgrading it in Thanos first.

Reply:
:+1: